docs/static/filebeat-modules.asciidoc

```diff
@@ -0,0 +1,321 @@
+[[filebeat-modules]]
+
+== Working with Filebeat Modules
+
+Starting with version 5.3, Filebeat comes packaged with pre-built 
+{filebeat}filebeat-modules.html[modules] that contain the configuration needed
```

Comment:
Myabe "configurations needed to collect, parse, enrich, and visualize data from various log file formats out-of-the-box"?